joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/astcreation/AstForTypesCreator.scala

```diff
@@ -154,9 +159,8 @@ trait AstForTypesCreator { this: AstCreator =>
         getClassNameScopedConstantReferenceContext(ctx.scopedConstantReference())
       else
         Option(ctx.CONSTANT_IDENTIFIER()).map(_.getText) match {
-          case Some(className) if baseClassName.isDefined => s"${baseClassName.get}.$className"
```

Comment:
@tuxology Can we not just remove this line, instead of splitting on `.`?

Reply:
We removed this line already. I can remove the splitting code as well - it was left there just in case.

Reply:
I believe If we just remove this line, this should fall in place. We should not need the splitting logic